.github/workflows/auto_release.yml

```diff
@@ -54,6 +54,6 @@ jobs:
       - name: Publish the release
         uses: softprops/action-gh-release@v1
         with:
-          tag_name: "v${{ needs.check_version.outputs.new_version }}"
+          tag_name: ${{ needs.check_version.outputs.new_version }}

```

Comment:
No worries my dude. Don't these need to match though? Looks like it's checking current_version with the v tag and setting the new one to have no v?

Reply:
Or do you mean just do this once and then we change it to just not have the v at all?

Reply:
No, right now both variables have "v" before the version number

Reply:
Ohh yes I see, my bad.